Styles/home.css

```diff
@@ -0,0 +1,154 @@
+/* Header */
+
+.header-auth-wrapper {
+    background-color: var(--white);
+    padding: var(--u-1);
+    margin-right: var(--u-2);
```

Comment:
please use generic names for your vars..
as u-1 u-2 will not be understood by others.